mica/common.py

```diff
@@ -8,7 +8,7 @@
 class MissingDataError(Exception):
     pass
 
-FLIGHT_MICA_ARCHIVE = '/proj/sot/ska/data/mica/archive'
+FLIGHT_MICA_ARCHIVE = os.path.join(os.environ['SKA'], 'data', 'mica', 'archive')
```

Comment:
I thought the whole point of the FLIGHT_MICA_ARCHIVE was that it was always actually the production mica archive, though I don't know if it matters.  Would it make just as much sense to set to None if not defined on your system (seems truer to the definition)?

Reply:
I'm not sure what is the point of hardwiring a "production" archive that does not exist on my machine.  And if I'm working in a non-production ska3 and have a dev version of the mica archive for in $SKA/data/mica, then the expected behavior is that it should pick up my local version by default.  It would be surprising to get the /proj/sot/ska version.

I guess I don't care about the internals, but  I do want to be able to get a starcheck catalog or dark cal without defining `MICA_ARCHIVE` in my bash profile.

Reply:
Sure.  When we set this up, the mica archive was basically the L0 image archive and the L1 solutions archive, we wouldn't plan to have those on a laptop, and the only reason for a non-production archive was for testing.  Use cases have changed.

Reply:
I'm not sure if if we should split the two, in fact, where the L0 archive and L1 archive stay in MICA_ARCHIVE but dark cal and starcheck go back into SKA/data/mica or the like.

Reply:
Also did you just delete the /proj/sot/ska/data/mica/archive link?  My annie second annie sim just died and now can't find dark cal data?

Reply:
In the unlikely event that it is necessary to have split access to different elements of the archive, it seems like using soft links from within $SKA/data/mica would do the trick.

Reply:
I was mostly thinking about setting up tests.  Right now we end up with different hacks to say "do you really have the relevant piece of the mica archive for this test" when running some tests, when really we mostly care about "do you have the big file mica archive" or do you have the "little useful mica archive" or do you have neither?  But that doesn't need to be an issue for this PR.  Just something to think about as an future enhancement.

Reply:
Should this be using ska_path for this instead of os.environ['SKA'] ?

Reply:
Doesn't really matter.  I don't like the current release functionality in ska_path.  I see that https://github.com/sot/ska_path/pull/4 never got merged.  With that, then the only advantage of using ska_path is a better error message if you don't have SKA defined.  

But right now without that PR merged (for version 4.2) then I don't want ska_path.

Reply:
I just haven't even been setting SKA in my environments because the modules have been correctly getting defaults.

Reply:
There is no longer a unique default in Ska3 and SKA is a required env var for our core packages, so these packages should no longer assume `/proj/sot/ska` anywhere.  We will eventually clean things up so that things will fail for you without SKA.  :smile:

Reply:
BTW, I purposely moved data on my standalone platforms out of /proj/sot/ska to identify problems (or rather things that accidentally work) related to this legacy default.

Reply:
Right and this is why I also wanted to ask @mbaski  about this in the [other thread](https://github.com/sot/proseco/issues/45) so that we wouldn't implement changes that are contrary to what is already assumed by setup scripts on the Window side.